modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +32,182 @@ The repository must be accessible from the hub cluster and be defined as a sourc
+        numVfs: 8
+        priority: 10
         resourceName: du_fh
-    - fileName: SriovNetwork.yaml
-      policyName: "config-policy"
+
+    - fileName: SriovNetwork.yaml # wave 100
+      policyName: "group-du-sno-sriov-policy"
       metadata:
-        name: "sriov-nw-du-mh"
+        name: sriov-nw-du-mh
       spec:
         resourceName: du_mh
-        vlan: '{{hub fromConfigMap "ztp-site" "sriovdata" (printf "%s-du_mh-vlan" .ManagedClusterName) | toInt hub}}'
-    - fileName: SriovNetworkNodePolicy.yaml
-      policyName: "config-policy"
+        vlan: '{{hub fromConfigMap "" "site-data-configmap" (printf "%s-sriov-network-vlan-2" .ManagedClusterName) | toInt hub}}'
+
+    - fileName: SriovNetworkNodePolicy.yaml # wave 100
+      policyName: "group-du-sno-sriov-policy"
       metadata:
-        name: "sriov-nnp-du-mh"
+        name: sriov-nw-du-fh
       spec:
-        deviceType: vfio-pci
+        deviceType: netdevice
         isRdma: false
         nicSelector:
-          pfNames:
-          - '{{hub fromConfigMap "ztp-site" "sriovdata" (printf "%s-du_mh-pf" .ManagedClusterName)  hub}}'
-        numVfs: '{{hub fromConfigMap "ztp-site" "sriovdata" (printf "%s-du_mh-numVfs" .ManagedClusterName) | toInt hub}}'
-        priority: '{{hub fromConfigMap "ztp-site" "sriovdata" (printf "%s-du_mh-priority" .ManagedClusterName) | toInt hub}}'
-        resourceName: du_mh
+          pfNames: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-sriov-node-policy-pfNames-2" (index .ManagedClusterLabels "hardware-type")) | toLiteral hub}}'
+        numVfs: 8
+        priority: 10
+        resourceName: du_fh
 ----
++
+[NOTE]
+====
+To retrieve site-specific configuration values, use the `.ManagedClusterName` field. 
+This is a template context value set to the name of the target managed cluster.
+
+To retrieve group-specific configuration, use the `.ManagedClusterLabels` field. 
+This is a template context value set to the value of the managed cluster’s labels.
+====
 
 . Commit the site `PolicyGenTemplate` CR in Git and push to the Git repository that is monitored by the ArgoCD application.
 +
 [NOTE]
 ====
-Subsequent changes to the referenced `ConfigMap` CR are not automatically synced to the applied policies. You need to manually sync the new `ConfigMap` changes to update existing PolicyGenTemplate CRs. See "Syncing new ConfigMap changes to existing PolicyGenTemplate CRs".
+Subsequent changes to the referenced `ConfigMap` CR are not automatically synced to the applied policies. 
+You need to manually sync the new `ConfigMap` changes to update existing `PolicyGenTemplate` CRs. See "Syncing new ConfigMap changes to existing PolicyGenTemplate CRs".
+
+You can use the same `PolicyGenTemplate` CR for multiple clusters. 
+If there is a configuration change, then the only modifications you need to make are to the three `ConfigMap` objects that hold the configuration for each cluster and the labels of the managed clusters.
```

Comment:
```suggestion
If there is a configuration change, then the only modifications you need to make are to the `ConfigMap` objects that hold the configuration for each cluster and the labels of the managed clusters.
```
I wouldn't say `to the three ConfigMaps` because the change might have to be to just one ConfigMap or to 2 ConfigMaps. Similarly, the users might have a different number of ConfigMaps in their setup.